recipes/liblsl/all/conanfile.py

```diff
@@ -25,18 +26,6 @@ class LiblslConan(ConanFile):
         "fPIC": True,
     }
 
```

Comment:
```suggestion

    @property
    def _min_cppstd(self):
        return "14"

    @property
    def _compilers_minimum_version(self):
        return {
            "Visual Studio": "15",
            "msvc": "191",
            "gcc": "5",
            "clang": "3.5",
            "apple-clang": "10",
        }

```
liblsl requires C++14, so we need to validate which compiler is running. Otherwise, it may fail when building, but no clue about compiler limitations.